components/pd_client/src/client.rs

```diff
@@ -5,15 +5,14 @@ use std::sync::{Arc, RwLock};
 use std::thread;
 use std::time::{Duration, Instant};
 
-use futures::channel::{mpsc, oneshot};
-use futures::compat::{Compat, Future01CompatExt, Sink01CompatExt, Stream01CompatExt};
+use futures::channel::mpsc;
+use futures::compat::Future01CompatExt;
```

Comment:
What do we still need the compat trait for?

Reply:
The new tokio timer must be used in tokio runtime, and the timer in tokio runtime cannot customize the clock, so I think it's better to continue to use tokio-timer. This is the only place where futures 0.1 is needed